lang/cs/Org.Apache.REEF.Network/Group/CommonOperators/DefaultCommonOperatorNames.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Org.Apache.REEF.Network.Group.CommonOperators
```

Comment:
Could you please split default operators and pipeline changes into two separate PR? You need to create branches for each.